server/rooms.ts

```diff
@@ -63,6 +63,46 @@ interface UserTable {
 	[userid: string]: User;
 }
 
+interface RoomSettings {
+	aliases: string[];
+	auth: {[userid: string]: GroupSymbol} | null;
```

Comment:
```suggestion
	auth: {[userid: string]: GroupSymbol};
```

I thought you said you made this non-null?